src/highrl/envs/robot_env.py

```diff
@@ -314,46 +240,46 @@ def render(
 
         # Create viewer
         if self.viewer is None:
-            self.viewer = rendering.Viewer(self.width, self.height)
+            self.viewer = rendering.Viewer(self.cfg.width, self.cfg.height)
             self.transform = rendering.Transform()
             self.transform.set_scale(10, 10)
             self.transform.set_translation(128, 128)
             self.score_label = pyglet.text.Label(
                 "0000",
                 font_size=12,
```

Comment:
Change font size printed so that it is suitable
I think it should be about 5
This should be done for both `score_label` and `iteration_label`